pilz_control/test/acceptance_test_speed_monitoring.py

```diff
@@ -78,17 +79,20 @@ class AcceptancetestSpeedMonitoring(unittest.TestCase):
     """
 
     def setUp(self):
-        param_name = CONTROLLER_NS + _JOINT_NAMES_PARAMETER
+        controller_ns = rospy.get_param(_CONTROLLER_NS_PARAMETER, _DEFAULT_CONTROLLER_NS)
```

Comment:
You stated in the PR description that it is possible to execute the acceptance tests without real hardware. I tried the `acceptance_test_speed_monitoring.py` like you described. In other words, I executed:
```
roslaunch pilz_control robot_mock.launch
rosrun pilz_control acceptance_test_speed_monitoring.py
```
I got an error. For details see attached picture:
![error_acceptance_test_speed_monitoring py](https://user-images.githubusercontent.com/40654276/85846139-af7e5380-b7a5-11ea-821c-630d8a8b26f5.png)

I'm guessing the test is not intended to be used without `joint_states_speed_observer`? In other words, I always have to launch the `joint_states_speed_observer`:
```
roslaunch pilz_control joint_states_speed_observer.launch robot_name:=robot_mock
```

Reply:
Yes. I added a description to the .md-file.